include/pulsar/c/consumer_configuration.h

```diff
@@ -304,6 +304,28 @@ PULSAR_PUBLIC void pulsar_consumer_configuration_set_start_message_id_inclusive(
 PULSAR_PUBLIC int pulsar_consumer_configuration_is_start_message_id_inclusive(
     pulsar_consumer_configuration_t *consumer_configuration);
 
+/**
+ * Set batch receive policy.
+ *
+ * The default value: {maxNumMessage: -1, maxNumBytes: 10 * 1024 * 1024, timeoutMs: 100}
+ * @param consumer_configuration  the consumer conf object.
+ * @param maxNumMessage default: Max num message, if less than 0, it means no limit.
+ * @param maxNumBytes Max num bytes, if less than 0, it means no limit.
+ * @param timeoutMs If less than 0, it means no limit.
+ */
+PULSAR_PUBLIC void pulsar_consumer_configuration_set_batch_receive_policy(
+    pulsar_consumer_configuration_t *consumer_configuration, int maxNumMessage, long maxNumBytes,
+    long timeoutMs);
+
+PULSAR_PUBLIC int pulsar_consumer_configuration_get_batch_receive_policy_max_num_message(
+    pulsar_consumer_configuration_t *consumer_configuration);
+
+PULSAR_PUBLIC long pulsar_consumer_configuration_get_batch_receive_policy_max_num_bytes(
+    pulsar_consumer_configuration_t *consumer_configuration);
+
+PULSAR_PUBLIC long pulsar_consumer_configuration_get_batch_receive_policy_timeout_ms(
+    pulsar_consumer_configuration_t *consumer_configuration);
```

Comment:
It might be better to define a struct that represents the batch receive policy to avoid too many setters.

```c
typedef struct {
    int maxNumMessage;
    int maxNumBytes;
    long timeoutMs;
} pulsar_batch_receive_policy_t;
```

Reply:
With struct, do users need to manually delete that? 

```c
pulsar_batch_receive_policy_t  policy = {100, 100, 1000};
pulsar_consumer_configuration_set_batch_receive_policy(consumer, policy);

delete policy;

```

Reply:
No. Only the memory allocated from the heap needs to be deleted.

Reply:
BTW, since the size of `pulsar_batch_receive_policy_t` is greater than 8 bytes, it's better to pass the pointer to the function, i.e. the function accepts a const pointer argument.

Reply:
If passing a pointer, users need to allocate/new this struct and manually free memory. 

```c
    pulsar_consumer_batch_receive_policy_t *batch_receive_policy;
    batch_receive_policy =
        (pulsar_consumer_batch_receive_policy_t *)malloc(sizeof(pulsar_consumer_batch_receive_policy_t));
    batch_receive_policy->maxNumMessage = 100;
    batch_receive_policy->maxNumBytes = -1;
    batch_receive_policy->timeoutMs = -1;
    pulsar_consumer_configuration_set_batch_receive_policy(consumer_conf, batch_receive_policy);
    delete batch_receive_policy;
```

Is this user friendly in C?